### ISSUE TITLE
Improvements with DateTime in databrowser

### ISSUE THREAD

**Issue by marktani** [graphcool/console#539]
When I create a `DateTime` field in the playground or in a script, this is the behaviour:

```graphql
mutation {
  createMovie(
      releaseDate: "2016-02-02"
      title: "Movie"
    ) {
      releaseDate
    }
}
```

```json
{
  "data": {
    "createMovie": {
      "movie": {
        "releaseDate": "2016-02-02T00:00:00.000Z"
      }
    }
  }
}
```

This is really helpful when I want to create dates and don't care about the time, for example for birthdays or release dates as in this example.

Unfortunately this behaviour is currently not possible in the console, because the date time picker automatically takes my timezone into account and I cannot add custom strings.

So if I select the 2nd February 2016 in the date time picker, this is the resulting ISO String shown in the date time picker: `2016-02-02 00:00:00+0100` which results in this ISO String stored in the backend: `2016-02-01T23:00:00.000Z`. 

(As a side note, an unselected cell in the data browser also takes my timezone into account, which adds another level of confusion because now the datetime looks to have the correct value: `2/2/2016, 12:00:00 AM`. Maybe we could even use ISO format here to keep it consistent.)

To summarize, I want the ability in the data browser to specify dates that end up in this format in my project's backend: `2016-02-02T00:00:00.000Z`. Currently this is only possible when having the UTC timezone locally.
